resources/metadata/README.md

```diff
@@ -0,0 +1,22 @@
+# Shuttle Metadata
+
+This plugin allows applications to obtain certain information about their runtime environment.
+
+## Usage
+
+Add `shuttle-metadata` to the dependencies for your service.
+
+You can get this resource using the `shuttle_metadata::ShuttleServiceInfo` attribute to get a `ServiceInfo`. This struct will contain information such as the Shuttle service name.
+
+```rust
+#[shuttle_runtime::main]
+async fn app(
+    #[shuttle_metadata::ShuttleServiceInfo] service_info: shuttle_metadata::ServiceInfo,
```

Comment:
We can also rename the structs

Reply:
done in 1c2aff749a266e2f5cbfa91f296b933822b6e97b